modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/login-button/index.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+const loginButton = document.getElementById('evp-login-button');
```

Comment:
Please, change the document. to fragmentElement. Because fragmentElement works with the fragment scope.
```suggestion
const loginButton = fragmentElement.getElementById('evp-login-button');
```

Reply:
Sure. Adjusted!